docs/sphinx/reference-libobs-util-serializers.rst

```diff
@@ -171,3 +171,44 @@ File Output Serializer Functions
 .. function:: void file_output_serializer_free(struct serializer *s)
 
    Frees the file output serializer and saves the file.
+
+---------------------
+
+Buffered File Output Serializer
+===============================
+
+Provides a buffered file serializer that writes data asynchronously to prevent stalls due to slow disk I/O.
+
+Writes will only block when the buffer is full.
+
+The buffer and chunk size are configurable with the defaults being 256 MiB and 1 MiB respectively.
+
+.. versionadded:: 30.2
+
+.. code:: cpp
+
+   #include <util/buffered-file-serializer.h>
+
+
+File Output Serializer Functions
```

Comment:
Should this read "Buffered File Output Serializer Functions" to distinguish it from the sub-heading in the section above this one?